bundle/src/main/java/com/adobe/acs/commons/http/JSONResponseHandler.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Any chance you can reformat using the projects formatting? It looks like youre at 80 char breaks which makes it rather hard to read.